src/js/pouch.js

```diff
@@ -0,0 +1,70 @@
+// Utility functions to add pouch to an existing express instance
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.pouch");
+
+var PouchDB        = require("pouchdb");
+var memdown        = require("memdown");
+var expressPouchdb = require("express-pouchdb");
+
+gpii.pouch.init = function(that) {
+    var MemPouchDB = PouchDB.defaults({db: memdown });
+
+    if (that.model.databases && Object.keys(that.model.databases).length > 0) {
+        Object.keys(that.model.databases).forEach(function(key){
+            var dbConfig = that.model.databases[key];
+            var db = new MemPouchDB(key);
+            if (dbConfig.data) {
+                var data = require(dbConfig.data);
+                db.bulkDocs(data);
+            }
+        });
+    }
+
+    that.expressPouchdb = expressPouchdb(MemPouchDB);
+
+    // Sadly, expressPouchdb does not provide a ready event or callback.  We have to manually enforce a waiting period by waiting to fire our "started" event.
+    setTimeout(function() { that.events.started.fire(); },500);
```

Comment:
@amb26, the tests pass when `started` is fired immediately. I will need to test with a larger data set, I suspect the problems that caused me to add this line are only apparently when the async call to db.bulkDocs() takes too long.  If I am able to demonstrate this, my suggested fix would be to make a chain of promises and only fire the event when all are complete.  If there is a way to do this with just the framework without using when.js, I would appreciate hearing about it.